src/Uno.UI/UI/Xaml/Controls/ListViewBase/VirtualizingPanelLayout.Android.cs

```diff
@@ -2677,6 +2685,45 @@ public VirtualizingPanelSmoothScroller(VirtualizingPanelLayout layout, RecyclerV
 
 			public override PointF ComputeScrollVectorForPosition(int targetPosition) => _layout.ComputeScrollVectorForPosition(targetPosition);
 
+			protected override void UpdateActionForInterimTarget(Action action)
+			{
+				// find an interim target position
+				var scrollVector = ComputeScrollVectorForPosition(TargetPosition); // direction only, not magniture WHERE x and y are in {-1,0,1}
+				if (scrollVector == null || (scrollVector.X == 0 && scrollVector.Y == 0))
+				{
+					var target = TargetPosition;
+					action.JumpTo(target);
+					Stop();
+					return;
+				}
+
+				Normalize(scrollVector);
+				MTargetVector = scrollVector;
+				MInterimTargetDx = (int)(TARGET_SEEK_SCROLL_DISTANCE_PX * scrollVector.X);
+				MInterimTargetDy = (int)(TARGET_SEEK_SCROLL_DISTANCE_PX * scrollVector.Y);
+
+				var extend = _layout.ComputeVerticalScrollExtent(_state);
+				MInterimTargetDx = Math.Min(MInterimTargetDx, extend);
+				MInterimTargetDy = Math.Min(MInterimTargetDy, extend);
+
+				var time = CalculateTimeForScrolling(extend);
+
+				// To avoid UI hiccups, trigger a smooth scroll to a distance little further than the
+				// interim target. Since we track the distance travelled in onSeekTargetStep callback, it
+				// won't actually scroll more than what we need.
```

Comment:
```suggestion
				// To avoid UI hiccups, trigger a smooth scroll to a distance a little further than the
				// interim target. Since we track the distance traveled in the onSeekTargetStep callback, it
				// won't actually scroll more than what we need.
```